packages/test/src/index.ts

```diff
@@ -1,4 +1,5 @@
-// @ts-ignore
+import type MatchMediaMock from "vitest-matchmedia-mock"
+export { MatchMediaMock }
```

Comment:
@hirotomoyamada 
are these required?

Reply:
@nakayan5 

This is required for type definitions in external source code.